examples/graphql-service-basic-auth-ldap-user-store/graphql_service_basic_auth_ldap_user_store.server.out

```diff
@@ -1,5 +1,3 @@
-# To run this sample, navigate to the directory that contains the `.bal` file,
-# and execute the `bal run` command below.
 # (You may need to change the certificate file path and private key file path.)
```

Comment:
Do we need these brackets anymore? Since this is the only instruction here, we can remove the bracket, right? Check the other place as well.

Reply:
Agree. Fixed with https://github.com/ballerina-platform/ballerina-distribution/pull/1886/commits/f3fca78c22aaa78dbea2491491bc265438df6fce